common/web/types/build.sh

```diff
@@ -82,9 +82,16 @@ function do_configure() {
 function do_test() {
   eslint .
   tsc --build test
-  readonly C8_THRESHOLD=50
-  c8 -skip-full --reporter=lcov --reporter=text --lines $C8_THRESHOLD --statements $C8_THRESHOLD --branches $C8_THRESHOLD --functions $C8_THRESHOLD mocha "${builder_extra_params[@]}"
-  builder_echo warning "Coverage thresholds are currently $C8_THRESHOLD%, which is lower than ideal."
+
+  readonly C8_THRESHOLD=60
+
+  # Excludes are defined in .c8rc.json
+  c8 --skip-full --reporter=lcov --reporter=text --lines "${C8_THRESHOLD}" \
+    --statements "${C8_THRESHOLD}" --branches "${C8_THRESHOLD}" \
+    --functions "${C8_THRESHOLD}" \
```

Comment:
I think this is an example of where linters are just generating busywork!  `$C8_THRESHOLD` --> `"${C8_THRESHOLD}"` ... given we can see the definition immediately above.

Reply:
reverted